content/css/concepts/background/terms/background-origin/background-origin.md

```diff
@@ -37,35 +37,47 @@ Set background positioning to the edge of the content box:
 
 ```css
 .main {
-  border: 5px dashed #f1f1f1;
-  padding: 15px;
-  background: linear-gradient(purple, yellow) no-repeat;
+  border: 5px dashed #4d4d4d;
+  padding: 1rem;
+  background: url(./media/background-origin.jpg) no-repeat;
```

Comment:
The css needs to call an image, but it doesn't have to be included in the media file. We only need the result of css, not the elements. And it doesn't have to use our path either, it could be simply -
background-origin.jpg 
or some other fake example directory like
images/background-origin.jpg